pkg/scheduler/extender/extender.go

```diff
@@ -594,3 +609,7 @@ func (e *Extender) scNameStorageTypeMapping(ctx context.Context) (map[string]str
 	}
 	return scNameTypeMap, nil
 }
+
+func generateEphemeralVolumeName(podName, volumeName string) string {
```

Comment:
What the point of separate function ? Just use `fmt.Sprintf("%v-%v", podName, volume.Name)`

Reply:
We should have the same names during both ephemeral and generic ephemeral scenario. And for me, `requestName := generateEphemeralVolumeName(podName, volume.Name)` looks better, it marks that the whole name is not just a Volume name

Reply:
But it implies from the code, this code tell you that the string is `podName - volume.Name` , there is no need of separate function for string formatting.

Reply:
In my opinion, it's not clear from the code, why we don't use VolumeName as it works with PVC. I've spent some time when just putting `v.Name` here and then decided to create a specific function with `EphemeralVolumes` in signature

Reply:
Go compiler should inline such simple functions, so it's just formatting question